default.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs ? import <nixpkgs> {} }:
+{
+  # The `lib`, `modules`, and `overlay` names are special
+  lib = import ./lib { inherit pkgs; }; # functions
+  modules = import ./modules; # NixOS modules
+  overlays = import ./overlays; # nixpkgs overlays
```

Comment:
The NUR template says that lib, modules and overlays are optional. Since we don't seem to use them wouldn't it be better to remove them?

Reply:
I removed lib and overlays. I would retain "modules" given that things inside of the modules/ dir are actual NixOS modules. Also that's where I think most of the value comes from, as packaging is one thing, but correctly wiring everything up is another :)